### PR TITLE
#151168135 Re-Design UI for user authentication screen

### PR DESCRIPTION
**Pull request by enodi**
**What does this PR do?**
Redesign of user signup and signin page

**Description of Task to be completed?**
Users should be able to see a redesigned signup/signin page with option to authenticate via twitter, github, facebook and google

**What are the relevant pivotal tracker stories?**
#151168135 - Re-Design UI for user authentication screen

**Screenshots (if appropriate)**
image

### CODE REVIEW

**Review by ibonly — CHANGES_REQUESTED** [review]
More than 5 files changed and just 2 commit messages, how come?

File: app/views/includes/head.jade [hunk 1]
```diff
@@ -23,9 +26,9 @@ head
   meta(property='og:site_name', content='Cards for Humanity')
   meta(property='fb:admins', content='APP_ADMIN')
 
-  link(rel='stylesheet', href='/lib/bootstrap/css/bootstrap.css')
-  link(rel='stylesheet', href='/css/common.css')
-  link(rel='stylesheet', href='/css/animate.css')
+  //- link(rel='stylesheet', href='/lib/bootstrap/css/bootstrap.css')
+  //- link(rel='stylesheet', href='/css/common.css')
+  //- link(rel='stylesheet', href='/css/animate.css')
```
Comment (gbengaoyetade) [inline app/views/includes/head.jade]: Remove unused code

File: server.js [hunk 2]
```diff
@@ -26,6 +26,7 @@ const env = process.env.NODE_ENV = process.env.NODE_ENV || 'development',
 
 // Bootstrap db connection
 const db = mongoose.connect(config.db);
+// const db = mongoose.connect('mongodb://localhost/rainier-cfh');
```
Comment (gbengaoyetade) [inline server.js]: Remove redundant code

File: app/controllers/users.js [hunk 3]
```diff
@@ -5,9 +5,10 @@
 const jwt = require('jsonwebtoken');
 const bcrypt = require('bcryptjs');
 const mongoose = require('mongoose');
-const dotenv = require('dotenv');
+require('dotenv').config();
+// const dotenv = require('dotenv');
```
Comment (gbengaoyetade) [inline app/controllers/users.js]: remove redundant code

**Review by gbengaoyetade — CHANGES_REQUESTED** [review]

**Review by dinobi — APPROVED** [review]
Seen

**Review by gbengaoyetade — APPROVED** [review]

**Review by ibonly — APPROVED** [review]

**Pull request merged**